Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM centos:6
+
+MAINTAINER IoT team
+
+RUN yum update -y && yum install -y wget
+RUN wget http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
+RUN yum localinstall -y --nogpgcheck epel-release-6-8.noarch.rpm
+RUN yum install -y rpm-build git java-1.7.0-openjdk java-1.7.0-openjdk-devel
```

Comment:
You may want to group several RUN commands into one. This helps speedup the build process and takes less space.